manifests/init.pp

```diff
@@ -61,13 +61,24 @@
     if ($uid) {
       validate_integer($uid)
     }
-    if ($manage_groups) {
-      $managed_groups = [$groups, $gid]
+
+    case $manage_groups {
+      true:  {
+        $managed_groups = [$groups, $gid]
+      }
+      'primary','gid':  {
+        $managed_groups = [$gid]
+      }
+      default: { }
```

Comment:
`false` and `'disabled'` should create an empty array, and other values should generate an error with `fail()`.